### ISSUE TITLE
Next.js & amp & react-jss - amp-custom attribute gets striped from the <style> tag

### ISSUE THREAD

**Issue by cibulka** [cssinjs/jss#1339]
Hello! 

I am not sure if what I'm about to describe is Next.js bug or `react-jss`'s, but here we go:

__Describe the bug:__
I'm trying to use [react-jss](https://cssinjs.org/react-jss/?v=v10.1.1) with Next.js as described in [Next.js official example](https://github.com/zeit/next.js/tree/master/examples/with-react-jss). However if I try to use it with Next.js amp build configuration, `amp-custom` attribute gets striped from the `<style>` element.

I've created issue about it here: https://github.com/zeit/next.js/issues/12243. 

__Expected behavior:__
`amp-custom` attribute should not get striped from `<style>` tag in `<head>`.

__Codesandbox link:__ 
As the issue is connected to Next.js, I couldn't really create a Sandbox link. Instead I've created repository with 2 Next.js projects (and different implementations of `react-jss`). Repo is here: Repo is here: https://github.com/cibulka/nextjs-amp-jss

__Versions (please complete the following information):__
 - jss: 10.1.1
 - Browser: Chrome
 - OS: macOS (10.15.4)

I'm only just starting with AMP, so I might be missing something obvious. Thank you!

**Comment by kof** [comment]
You are removing the servers-side generated style element yourself - https://github.com/cibulka/nextjs-amp-jss/blob/7d380da9c4487cfcb68e42fb881d984c110753bd/jss-example/pages/_app.js#L8

If you need to keep the attribute, you would have to add it when jss renderes styles on the client. JSS doesn't have an option for adding attributes to the style element yet.

Maybe you could add the attribute after the first render by querying all style elements created by jss? It's a probelm if it needs to always exist for style elements rendered later on though.

**Comment by cibulka** [comment]
Of course, that makes sense. I'm not working with AMP at the moment (my issue was more then a year old) so I can't provide a solution, but maybe it will be helpful for somebody else. Thanks!
